rotkehlchen/tests/api/test_assets.py

```diff
@@ -70,6 +70,18 @@ def assert_substring_in_search_result(
         assert substr_in_name or substr_in_symbol, f'no match for {substring}'
 
 
+def assert_asset_at_top_position(
+        asset_name: str,
+        max_position_index: int,
+        result: List[Dict[str, Any]],
+) -> None:
+    """Aserts that an asset appears at the top of the search results."""
+    assert any([asset_name == entry['name'] for entry in result])
```

Comment:
no no no, you should check the identifier and not the name.Name can be anything. The only thing that identifies bitcoin as bitcoin in rotki is `identifier=='BTC'`, ethereum as `ETH` etc.

Reply:
Such tests are already present(same answer as below).
https://github.com/rotki/rotki/blob/894f8c68147289991151b3a6189e05a46dba39bd/rotkehlchen/tests/api/test_assets.py#L463

https://github.com/rotki/rotki/blob/894f8c68147289991151b3a6189e05a46dba39bd/rotkehlchen/tests/api/test_assets.py#L611

They both contain tests that check behaviours when treat eth2 as eth is on and off.